applications/mne_analyze/plugins/dataloader/dataloader.cpp

```diff
@@ -259,21 +262,55 @@ void DataLoader::onLoadFilePressed()
     //Get the path
     QString sFilePath = QFileDialog::getOpenFileName(Q_NULLPTR,
                                                     tr("Open File"),
-                                                    QDir::currentPath()+"/MNE-sample-data",
+                                                    m_sLastDir,
                                                     tr("Fiff file (*.fif *.fiff);;Event file (*.eve)"));
 
     QFileInfo fileInfo(sFilePath);
 
-    if(fileInfo.fileName().isEmpty()){
+    if(fileInfo.fileName().isEmpty() || !fileInfo.isFile()){
+
         return;
     }
 
+    updateLastDir(fileInfo.absolutePath());
+
     loadFilePath(sFilePath);
     #endif
 }
 
 //=============================================================================================================
 
+void DataLoader::updateLastDir(const QString& lastDir)
+{
+    m_sLastDir = lastDir;
+    saveSettings();
+}
+
+//=============================================================================================================
+
+void DataLoader::saveSettings()
+{
+    QSettings settings("MNECPP");
```

Comment:
Please use `MNEANALYZE/DATALOADER` instead of `MNECPP`

Reply:
This "MNECPP" is the top key value under which every setting/data is stored in the win register(win)/config files(unix). It is used in mainwindow, several other plugins, etc...